### ISSUE TITLE
Won’t Format in VS Code 1.23.0

### ISSUE THREAD

**Issue by alexclst** [kokororin/vscode-phpfmt#27]
I cannot get phpfmt to format code at all right now. It does not show up in the contextual menu, nor does it format on save any longer. I can’t seem to find any errors in the Developer Tools console that indicate problems, but my guess is that the latest update to VS Code (1.23.0) may have broken this extension. I’ve tried disabling phpfmt, restarting VS Code, reenabling it, and so forth. Nothing has gotten it to work again.

**Comment by kokororin** [comment]
It works in 1.23.0 and 1.23.1. You can see the screenshot
![123](https://user-images.githubusercontent.com/10093992/39900972-0b9299dc-54f8-11e8-9d02-3e32e3094d41.gif)

**Comment by starjou** [comment]
Not work here too.
If I execute from F1, it says command 'phpfmt.format' not found.
Execute by shortcut. says there is no php formater.
I downgrade to 1.22, 1.21, still same.

**Comment by kokororin** [comment]
Try to re-enable other extension may work 
https://github.com/prettier/prettier-vscode/issues/447

**Comment by alexclst** [comment]
@kokororin, I notice that you’re using a different operating system from me. I’m on a Mac, as is my colleague who is experiencing the same thing. Maybe there is something breaking phpfmt on Macs but not on your operating system? I don’t know enough about Electron to know if that is plausible.

Anyhow, this evening VS Code updated itself to 1.23.1, and I then tried fully uninstalling the extension, and then upon reinstalling phpfmt and enabling it I see errors in the developer console related to it (the first three, the fourth occurred even with phpfmt uninstalled):

![screenshot 2018-05-10 at 9 44 19 pm](https://user-images.githubusercontent.com/2946567/39904109-f852605c-549b-11e8-811b-755dbc21eae7.jpg)

So perhaps the problem rests somewhere in `detect_indent` or related extension APIs. I know nothing about how VS Code extensions are built, but maybe these error messages could help you fix the problem. These console log entries do make it seem like there is a real problem with this extension’s interaction with VS Code on my Mac, not anything related to how my system is configured (that I did intentionally at least).
